api/events/lambda_function.py

```diff
@@ -55,7 +55,12 @@ def lambda_handler(event, _):
         if event["source"] == "aws.medialive":
             # handle medialive pipeline alerts
             if "MediaLive" in event["detail-type"] and "Alert" in event["detail-type"]:
-                event["resource_arn"] = event["detail"]["channel_arn"]
+                # multiplex pipeline alerts do not have a "detail.channel_arn" property.
+                if "Multiplex" in event["detail-type"]:
+                    event["resource_arn"] = event["detail"]["multiplex_arn"]
+                    item["resource_arn"] = event["detail"]["multiplex_arn"]
+                else:
+                    event["resource_arn"] = event["detail"]["channel_arn"]
```

Comment:
Without evaluating whether this is from a channel or multiplex, we would get a python error on all multiplex alerts like the following:
```
    'channel_arn': KeyError
    Traceback (most recent call last):
      File "/var/task/lambda_function.py", line 58, in lambda_handler
        event["resource_arn"] = event["detail"]["channel_arn"]
    KeyError: 'channel_arn'
```